### PR TITLE
HP-483 Remove keycloak client

### PR DESCRIPTION
**Pull request by NikoHelle**
Removed keycloak.js specific code and kept only oidc-react.js as only client library.  Kept "client/index.ts" as client wrapper, because supporting Tunnistamo, Keycloak and Helsinki Profiili is then simpler. It also simplifies hooks.

Renamed "/clients" to "/client", which increased "files changed" count a lot. There are only a few "real" changes outside of "client"-folder.

Tunnistamo is now the default OIDC provider in .env -files.

### CODE REVIEW

File: src/client/__tests__/client.test.ts [hunk 1]
```diff
@@ -0,0 +1,254 @@
+// following ts-ignore + eslint-disable fixes "Could not find declaration file for module" error for await-handler
```
Comment (NikoHelle) [inline src/client/__tests__/client.test.ts:1]: This is not really a 100% changed file. Removed array wrapper around tests which probably caused this.

File: README.md [hunk 2]
```diff
@@ -1,85 +1,41 @@
...
-REACT_APP_OIDC_URL="https://api.hel.fi/sso-test"
-REACT_APP_OIDC_CLIENT_ID="https://api.hel.fi/auth/helsinkiprofile-ui-dev"
-REACT_APP_OIDC_SILENT_AUTH_PATH="/callback"
-REACT_APP_OIDC_AUTO_SIGN_IN=false
-REACT_APP_OIDC_SCOPE="openid profile https://api.hel.fi/auth/helsinkiprofiledev"
-REACT_APP_OIDC_CLIENT_TYPE="oidc"
-REACT_APP_OIDC_REALM=""
-REACT_APP_OIDC_CALLBACK_PATH="/callback"
-REACT_APP_OIDC_TOKEN_EXCHANGE_PATH="/api-tokens/"
+REACT_APP_OIDC_URL="https://helsinki-profile-keycloak-dev.agw.arodevtest.hel.fi/auth"
```
Comment (akikoskinen) [inline README.md:33]: I wouldn't put this temporary URL here. Perhaps write "TODO" here or some imaginary URL like "https://your-keycloak-server.example/auth".

**Review by akikoskinen — CHANGES_REQUESTED** [review]
Nothing major, just a few small things.

File: README.md (hunk 2 above)
Comment (NikoHelle) [reply README.md]: Changed to <KEYCLOAK_SERVER_URL>/auth

File: src/pages/Index.tsx [hunk 3]
```diff
@@ -25,16 +23,15 @@ const Index = (): React.ReactElement => {
           </p>
           <p>Voit myös kirjatua ulos toisessa ikkunassa.</p>
           <p>
-            Clientiksi on .env-filessä määritelty:
-            <strong>{clientConfig.type}</strong>
+            Client on: <strong>Oidc</strong>
           </p>
```
[the remark this answers is not in the document]
Comment (NikoHelle) [reply src/pages/Index.tsx]: Correct, this is useless info now.

**Review by akikoskinen — APPROVED** [review]

**Pull request merged**